nilearn/plotting/html_surface.py

```diff
@@ -0,0 +1,386 @@
+
+    threshold : str, number or None, optional (default=None)
+        If None, no thresholding.
+        If it is a number only values of amplitude greater
+        than threshold will be shown.
+        If it is a string it must finish with a percent sign,
+        e.g. "25.3%", and only values of amplitude above the
+        given percentile will be shown.
+
+    cmap : str or matplotlib colormap, optional
+
+    black_bg : bool, optional (default=False)
+
+    embed_js : bool, optional (default=True)
+        if True, jquery and plotly are embedded in resulting page.
+        otherwise, they are loaded via CDNs.
+
+    Returns
+    -------
+    SurfaceView : plot of the stat map.
+        It can be saved as an html page or rendered (transparently) by the
+        Jupyter notebook.
+
+    """
+    info = full_brain_info(
+        volume_img=volume_img, mesh=mesh, threshold=threshold,
+        cmap=cmap, black_bg=black_bg)
+    return _fill_html_template(info, embed_js=embed_js)
+
+
+def view_surf(surf_mesh, surf_map=None, bg_map=None, threshold=None,
+              cmap=cm.cold_hot, black_bg=False, symmetric_cmap=True,
+              embed_js=True):
+    """
+    Insert a surface plot of a surface map into an HTML page.
+
+    Parameters
+    ----------
+    surf_mesh: str or list of two numpy.ndarray
+        Surface mesh geometry, can be a file (valid formats are
+        .gii or Freesurfer specific files such as .orig, .pial,
+        .sphere, .white, .inflated) or
+        a list of two Numpy arrays, the first containing the x-y-z coordinates
+        of the mesh vertices, the second containing the indices
+        (into coords) of the mesh faces.
+
+    surf_map: str or numpy.ndarray, optional.
+        Data to be displayed on the surface mesh. Can be a file (valid formats
+        are .gii, .mgz, .nii, .nii.gz, or Freesurfer specific files such as
+        .thickness, .curv, .sulc, .annot, .label) or
+        a Numpy array
+
+    bg_map: Surface data, optional,
+        Background image to be plotted on the mesh underneath the
+        surf_data in greyscale, most likely a sulcal depth map for
+        realistic shading.
+
+    threshold : str, number or None, optional (default=None)
+        If None, no thresholding.
+        If it is a number only values of amplitude greater
+        than threshold will be shown.
+        If it is a string it must finish with a percent sign,
+        e.g. "25.3%", and only values of amplitude above the
+        given percentile will be shown.
+
+    cmap : str or matplotlib colormap, optional
+        You might want to change it to 'gnist_ncar' if plotting a
+        surface atlas.
+
+    black_bg : bool, optional (default=False)
+
+    symmetric_cmap : bool, optional (default=True)
+        Make colormap symmetric (ranging from -vmax to vmax).
+        Set it to False if you are plotting a surface atlas.
+
+    embed_js : bool, optional (default=True)
```

Comment:
Is this necessary ? Will it not be there by default ?

I am thinking of reducing the cost of maintenance here.